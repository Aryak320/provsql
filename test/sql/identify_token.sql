\set ECHO none
\pset format unaligned
SET search_path TO provsql_test,provsql;

CREATE TABLE identify_result AS
SELECT identify_token(provenance())::text FROM personnel;

SELECT remove_provenance('identify_result');
SELECT * FROM identify_result;
DROP TABLE identify_result;
