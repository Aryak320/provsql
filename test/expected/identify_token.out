\set ECHO none
remove_provenance

(1 row)
identify_token
(personnel,5)
(personnel,5)
(personnel,5)
(personnel,5)
(personnel,5)
(personnel,5)
(personnel,5)
(7 rows)
